docs/modules/user/pages/try/15_minutes.adoc

```diff
@@ -0,0 +1,229 @@
+= Try James in 15 minutes
+
+In this tutorial, we will set up a runing James demo using a prepared Docker image.
+We will then test the server by connecting with an email client. Finally, we
+will connect to the server via the REST-based Admin API.
+
+
+Requirements: 
+
+ * Docker
+ * curl
+ * Thunderbird client (optional)
+
+== Set up the demo server
+
+Before starting the server, because we will be connecting to the Admin API we need to set some configuration parameters.
+We will create a `webadmin.properties` file that James will use in order to allow us to connect to the API.
+
+Run this command to create the `webadmin.properties` file:
+
+[source,bash]
+----
+printf 'enabled=true\nport=8000\nhost=localhost' >> webadmin.properties
+----
+
+Explanation:
+
+ * `enabled=true` instructs James to run the Admin API service
+ * `port=8000` configures the Admin API to be made available via port 8000
+ * `host=localhost` configures the Admin API to respond on localhost
+
+
+Now run the James demo server using this command:
+
+[source,bash]
+----
+docker run -d -p "25:25" -p "143:143" -p "8000:8000" -v $(pwd)/webadmin.properties:/root/conf/webadmin.properties --name james linagora/james-jpa-sample:3.4.0
+----
+
+Explanation:
+
+ * `docker run` runs the provided image with the given parameters
+ * The `-d` parameter runs the container in "detached" mode
+ * `-p "25:25" -p "143:143"` attaches the image ports to the ports 25 (SMTP) and 143 (IMAP) on the host machine
+ * `-p "8000:8000" attaches the port 8000 to the host machine to allow us to connect to the Admin API
+ * `-v $(pwd)/webadmin.properties:/root/conf/webadmin.properties` mounts the webadmin.properties file to configure the Admin API
+ * The `--name james` parameter gives the running container a name to make it easier to manipulate
+ * `linagora/james-jpa-sample:3.4.0` is the image that is used for this demo
+
+Docker will pull the image and start the container.
+
+****
+To run commands using the James CLI, you can use the running container via Docker:
+
+----
+docker exec james java -jar /root/james-cli.jar \
+    -h \<<HOST>> -p \<<PORT>> \<<COMMAND>>
+----
+
+In this tutorial, we are using host 127.0.0.1 and port 9999, so every command looks like:
+
+----
+docker exec james java -jar /root/james-cli.jar \
+    -h 127.0.0.1 -p 9999 \<<COMMAND>>
+----
+
+Host 127.0.0.1 is of course localhost, and the use of port 9999 is completely arbitrary.
+
+To make this tutorial a little easier, set this up as a bash script by copying and pasting this script:
+
+[source,bash]
+----
+printf '#!/bin/bash\n\ndocker exec james java -jar /root/james-cli.jar -h 127.0.0.1 -p 9999 $@' >> james ; chmod +x james
+----
+****
+
+As an example, list all the domains currently in use by James:
+
+[source,bash]
+----
+./james listdomains
```

Comment:
How about adding this little bash script in the docker image? Looks convenient...

This would turn the command into `docker exec james james-cli addDomain blah blah` which should be easy enough for a 15 minutes tutorial...

Reply:
Great! Yes!

Reply:
This will be captured as an issue elsewhere, so I am resolving here.